web/src/enterprise/campaigns/global/marketing/CampaignsMarketing.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+export interface CampaignsMarketingProps {
+    body: JSX.Element
+}
+
+export const CampaignsMarketing: React.FunctionComponent<CampaignsMarketingProps> = ({ body }) => (
+    <>
+        <section className="mt-3 mb-5">
+            <h1 className="font-weight-bold display-4">
+                Campaigns <span className="badge badge-info badge-outline">Beta</span>
+            </h1>
+            <p className="lead">
+                <em>
+                    Campaigns are currently in beta: during the beta period Campaigns are free to use, after the beta
+                    period Campaigns will be a paid add-on. Please <a href="mailto:sales@sourcegraph.com">contact us</a>{' '}
+                    for more information.
+                </em>
+            </p>
+            <h2 className="mb-6">Make large-scale code changes across all your repositories and code hosts.</h2>
+            <p className="mt-3">
+                <a href="https://about.sourcegraph.com/product/code-change-management" rel="noopener">
+                    Learn how
+                </a>{' '}
+                campaigns can help you remove legacy code, fix critical security issues, and pay down tech debt. See it
+                in action to run <code>go mod tidy</code> over all Go repositories:
```

Comment:
```suggestion
                in action to run <code>gofmt</code> over all Go repositories:
```

The teaser image of the video is wrong.